igamt-lite-client/app/scripts/clone/clone.service.js

```diff
@@ -333,6 +333,9 @@ angular.module('igl').factory(
 
         if(table.scope=='PHINVADS'){
           newTable.referenceUrl= $rootScope.getPhinvadsURL(table);
+          if($rootScope.tableLibrary.codePresence[table.id]!==false &&table.numberOfCodes<500){
```

Comment:
Why are we checking this only for PHINVADS. I thought we've already agreed on this.

Reply:
Because I need this check to add Phinvad URL. I am doing it in the client because we never store this attribute in the object.